README.md

```diff
@@ -56,13 +56,13 @@ Use `grunt docs` to build documentation without setting a server.
 
 The `repos/` folder contains submodules for the repositories currently included in the API documentation. As expected, the API is documented inline in the source code contained in these repositories, and is then integrated into the documentation files.
 
-## Using Local Versions of ckeditor-dev
+## Using Local Versions of ckeditor4
 
-While the main CKEditor repository for API documents, [ckeditor-dev](https://github.com/ckeditor/ckeditor-dev), is available as a submodule, it is also possible to make the builder use its local copy to avoid submodule limitations and speed up API documentation work. There are two ways to achieve it:
+While the main CKEditor repository for API documents, [ckeditor4](https://github.com/ckeditor/ckeditor4), is available as a submodule, it is also possible to make the builder use its local copy to avoid submodule limitations and speed up API documentation work. There are two ways to achieve it:
 
- * Keeping `ckeditor-docs/` and `ckeditor-dev/` folders in the same directory.
+ * Keeping `ckeditor4-docs/` and `ckeditor4/` folders in the same directory.
 
- * Setting the `CKEDITOR_DEV_PATH` environment variable to point to your `ckeditor-dev/` folder path.
+ * Setting the `CKEDITOR_DEV_PATH` environment variable to point to your `ckeditor4/` folder path.
```

Comment:
Isn't the environment variable simply `CKEDITOR_DEV`, though? See https://github.com/ckeditor/ckeditor4-docs/blob/master/gruntfile.js#L149.

Reply:
It's a really good question. We're using `CKEDITOR_DEV` however we supposed to use `CKEDITOR_DEV_PATH` according to different sources. That's why we should either update Readme to reflect actual state, or actual a gruntfile to follow our convention.

Reply:
Ahhhhha! And I was wondering why I always get `CKEDITOR_DEV env variable not set.` info, even though I have correctly set all the paths mentioned in our `tools` instructions. I think it should be `CKEDITOR_DEV_PATH` (it was `CKEDITOR_DEV` earlier but I'm not sure if it worked for anyone - most of use build based on folder structure so I think we could change it).